pkg/lockutil/lockutil_windows.go

```diff
@@ -33,20 +33,37 @@ var (
 	procUnlockFileEx = modkernel32.NewProc("UnlockFileEx")
 )
 
+const (
+	// see https://msdn.microsoft.com/en-us/library/windows/desktop/aa365203(v=vs.85).aspx
+	LOCKFILE_EXCLUSIVE_LOCK   = 0x00000002
+	LOCKFILE_FAIL_IMMEDIATELY = 0x00000001
```

Comment:
I used the same constant names used in windows docs to make it easy to find the flag in the docs or using search.

I see that the original code form bolt use nicer go style:
https://github.com/boltdb/bolt/blob/2f1ce7a837dcb8da3ec595b1dac9d0632f0f99e8/bolt_windows.go#L18

We can use this style.